torch/csrc/distributed/autograd/utils.cpp

```diff
@@ -31,6 +31,11 @@ void addSendRpcBackward(
     autogradContext.addSendFunction(
         grad_fn, autogradMetadata.autogradMessageId);
   }
+
+  // Record the workerID
+    if (dst.id_ != -1) {
+      autogradContext.addKnownWorkerID(dst);
+    }
```

Comment:
Indent mismatch. Run clang-format and never worry about this again ;-)

Reply:
I was trying to run clang-format with `python tools/clang_format.py`, and it didn't output anything so I thought there weren't any issues. It turns out that I should have checked this more thoroughly though, since the script does not appear to run `clang-format` on `torch/csrc/distributed` files. Is there a reason for this?

Reply:
Yeah, the script has gone stale (also see #25700). It's fine to run `clang-format -i` on the files yourself, because all files in `torch/csrc/distributed` and `torch/lib/c10d` should already be fully formatted (and you wouldn't tack on formatting changes unrelated to your commit).